src/Resources/config/services.yml

```diff
@@ -91,3 +101,37 @@ services:
     class: 'Flagbit\Bundle\CategoryBundle\Schema\SchemaValidator'
     arguments:
       - '../Resources/config/schema/config.json'
+
+  flagbit.category.data_transformer.normalization_transformer_manager:
+    class: 'Flagbit\Bundle\CategoryBundle\Transformer\NormalizationTransformerManager'
+    arguments:
+      - '@flagbit.category.repository.category_config'
+
+  flagbit.category.data_transformer.denormalization_transformer_manager:
+    class: 'Flagbit\Bundle\CategoryBundle\Transformer\DenormalizationTransformerManager'
+    arguments:
+      - '@flagbit.category.repository.category_config'
+
+  flagbit.category.data_transformer.denormalization.default:
+    class: 'Flagbit\Bundle\CategoryBundle\Transformer\Denormalization\DefaultTransformer'
+    tags:
+      - name: 'flagbit.category.data_transformer.denormalization'
+        type: 'default'
+
+  flagbit.category.data_transformer.normalization.default:
+    class: 'Flagbit\Bundle\CategoryBundle\Transformer\Normalization\DefaultTransformer'
+    tags:
+      - name: 'flagbit.category.data_transformer.normalization'
+        type: 'default'
+
+  flagbit.category.data_transformer.normalization.checkbox:
+    class: 'Flagbit\Bundle\CategoryBundle\Transformer\Normalization\BooleanTransformer'
+    tags:
+      - name: 'flagbit.category.data_transformer.normalization'
+        type: 'checkbox'
+
+  flagbit.category.data_transformer.denormalization.checkbox:
+    class: 'Flagbit\Bundle\CategoryBundle\Transformer\Denormalization\BooleanTransformer'
+    tags:
+      - name: 'flagbit.category.data_transformer.denormalization'
+        type: 'checkbox'
```

Comment:
Let's put the data transformders and their managers into a separate service file.